doc/specs/application-auth-token.md

```diff
@@ -20,26 +35,79 @@ A semver string specifying the spec version under which this AAT needs to be int
 
 > type: `string`
 
-The application will sign a hash of the `message` property within this token with the specified `appPubKey` and
-corresponding private key.
+The application will sign a hash of the `message` property within this token with
+the specified `appPubKey` and corresponding private key.
 
 ### applicationPublicKey
 
 > type: `string`
 
-The hexadecimal publicKey of the Application
+The hexadecimal `publicKey` of the `Application`
 
 ### clientPublicKey
 
 > type: `string`
 
-Required for signature verification, the hexadecimal public of each individual client allowing for granular control of
-who can use the AAT
+Required for signature verification.
+
+Also known as the `gatewayPublicKey`.
+
+The hexadecimal public key allowing granular control of who can use the `AAT`.
 
 ## ECDSA ed25519 Signature Scheme
 
-The protocol wide ed25519 ECDSA will be used for any signatures and verifications that are used within this
-specification.
+The protocol wide ed25519 ECDSA will be used for any signatures and verifications
+that are used within this specification.
+
+## AAT Uses-Cases
```

Comment:
I believe AAT has a single use, i.e. allow Gateways access Service nodes on behalf of the application. `Generating AAT` seems like a better title here, with the two cases of `Application === Client` and `Application !== Client` as the two different ways to generate an AAT. The current `AAT Generation` section title can be then merged with these two cases to capture the common steps between them.